### ISSUE TITLE
Build error: 'x' must be numeric

### ISSUE THREAD

**Issue by sneumann** [sneumann/mtbls2#17]
http://bioconductor.org/checkResults/3.11/data-experiment-LATEST/mtbls2/malbec2-buildsrc.html
```
Quitting from lines 189-228 (MTBLS2-xcms3.Rmd) 
Error: processing vignette 'MTBLS2-xcms3.Rmd' failed with diagnostics:
'x' must be numeric
--- failed re-building ‘MTBLS2-xcms3.Rmd’

--- re-building ‘MTBLS2.Rmd’ using rmarkdown
--- finished re-building ‘MTBLS2.Rmd’
```

**Comment by sneumann** [comment]
This should be fixed in xcms >= 3.9.4

**Comment by sneumann** [comment]
Fixed by XCMS 3.9.4
